modules/ROOT/pages/called-processes.adoc

```diff
@@ -33,7 +33,7 @@ Define the called process as you would define any other process. When you create
 In the parent process, define a call activity task. Define the normal task characteristics. Note that although a call activity cannot
 have forms associated with it, it can have connectors and task-level data.
 
-In the *Details* panel, *General* tab, *General* pane, specify the name of the called process. If you know the process name, you can enter it directly, otherwise use the expression editor.
+In the *Details* panel, *General* tab, *General* pane, specify the name of the called process. If you know the process name, you can enter it directly, otherwise use the expression editor. The version is optional, when not specified, the engine takes the last version deployed of the process.
```

Comment:
```suggestion
In the *Details* panel, *General* tab, *General* pane, specify the name of the called process. If you know the process name, you can enter it directly, otherwise use the expression editor. 

[NOTE]
====

The version is optional, when not specified, the engine takes the last version deployed of the process.
====
```